integration_tests/snaps/local-source-type/snapcraft.yaml

```diff
@@ -0,0 +1,12 @@
+name: test-package
+version: 0.1
+summary: one line summary
+description: a longer description
+icon: icon.png
```

Comment:
Please toast the icon, it's not necessary anymore (yeah I know, some of our older integration tests still have them, not your fault).